lib/beacon_api/router.ex

```diff
@@ -0,0 +1,29 @@
+defmodule BeaconApi.Router do
+  use BeaconApi, :router
+
+  pipeline :api do
+    plug(:accepts, ["json"])
+  end
+
+  # Ethereum API Version 1
+  scope "/eth/v1", BeaconApi.V1 do
+    pipe_through(:api)
+
+    scope "/beacon" do
+      get("/states/:state_id/root", BeaconController, :get_state_root)
+    end
+  end
+
+  # Ethereum API Version 2
```

Comment:
don't think we need this now

Reply:
We will need it to implement the v2 endpoints of the beacon api. 
Should I:
- Remove it for now and add back when we implement the endpoints
- Leave it empty for boilerplate so we know how and where to implement

Reply:
I would just keep the endpoints that we implement and remove boilerplate. I think it's trivial where to implement.

Reply:
Ok